workflow/cron/operator_test.go

```diff
@@ -27,7 +27,7 @@ var scheduledWf = `
     name: hello-world
     namespace: argo
     resourceVersion: "53389"
-    selfLink: /apis/argoproj.io/v1alpha1/namespaces/argo/cronworkflows/hello-world
+    selfLink: /apis/argoproj.io/v1alpha1/namespaces/argo/cronworkflow.hello-world
```

Comment:
Is this intentional?

Reply:
This just formatted this way, removing now, good catch.